src/components/ExploreGames/ExploreGames.jsx

```diff
@@ -0,0 +1,73 @@
+import { Card } from "shoto-ui";
+import "./exploreGames.css";
+
+export const ExploreGames = () => {
+
+  // Temporary function, to be updated in future version
+  const clickAction = () => {
+    //perform action
+    return;
+  }
+
+  return (
+    <div className="product-grid">
+      <Card

```

Comment:
These cards can use a  higher order `map` function too similar to how I have mentioned earlier.